docs/features/software-catalog/system-model.md

```diff
@@ -108,3 +108,26 @@ domain would come with some documentation on how to accept payments for a new
 product or use-case, share the same entity types in their APIs, and integrate
 well with each other. Other domains could be “Content Ingestion”, “Ads” or
 “Search”.
+
+### User
+
+A user describes a person, such as an employee, a contractor, or similar.
+
+### Group
+
+A group describes an organizational entity, such as for example a team, a
+business unit, or a loose collection of people in an interest group.
+
+### Location
+
+A location is a marker that references other places to look for catalog data.
+
+### Type
```

Comment:
Only thing I note here is the top-level heading structure. Note that we currently have `## Core Entities` and `## Ecosystem Modeling`. Maybe User and Group could go under a new `## Organizational Entities`, and then ... hm, type and template could be under a `## Other` 🤷

Reply:
The intention was to show a unified view of how the different pieces fit together, making it easier for a new adopter to understand the relationships between the objects (Ecosystem Modeling). 

What do you think about adding the sections you mention but also a graph that unifies all the pieces?